example/package.json

```diff
@@ -4,15 +4,18 @@
   "private": true,
   "scripts": {
     "android": "react-native run-android",
+    "clean:android": "rm -rf android/.gradle android/build android/.idea android/*.iml",
     "compile": "tsc -p .",
     "ios": "react-native run-ios",
     "lint": "eslint .",
-    "start": "node node_modules/react-native/local-cli/cli.js start --reset-cache",
+    "start": "node node_modules/react-native/local-cli/cli.js start --reset-cache --port 8082",
     "test": "jest"
   },
   "dependencies": {
+    "@react-native-clipboard/clipboard": "^1.6.0",
```

Comment:
already set as dependencies in netwatch, do we need to add it here too?

Reply:
Removed in feat/share